tests/UnitTests/Validator/UrlValidatorTest.php

```diff
@@ -91,6 +91,18 @@ public function correctUrl()
             yield [
                 'https://www.fb.io/some/path?a=1&b=14',
             ],
+            yield [
+                'ftp://google.com',
+            ],
+            yield [
+                'sftp://www.fb.io/some/path',
+            ],
+            yield [
+                'ws://192.168.1.15:8000',
+            ],
+            yield [
+                'wss://192.168.1.15:8000/messages',
+            ],
             yield [
                 '//yahoo.com',
             ],
```

Comment:
Please add some more tests on invalid values from https://d-mueller.de/blog/why-url-validation-with-filter_var-might-not-be-a-good-idea/